process/transaction/interceptor_test.go

```diff
@@ -28,7 +27,11 @@ func TestNewTxInterceptor_NilMarshalizerShouldErr(t *testing.T) {
 	addrConv := &mock.AddressConverterMock{}
 	keyGen := &mock.SingleSignKeyGenMock{}
 	oneSharder := mock.NewOneShardCoordinatorMock()
-	txValidator := &mock.TxValidatorStub{}
+	txValidator := &mock.TxValidatorStub{
+		GetNumRejectedTxsCalled: func() uint64 {
```

Comment:
you don't need in most of the cases to give an implementation to the GetNumRejectedTxsCalled because GetNumRejectedTxs is used only in a logger and in a test. please add it only where is needed

Reply:
this applies to the entire test file

Reply:
The implementation GetNumRejectedTxsCalled is needed, same test will return error because if a transaction is rejected  log.Debug will be called and there will be an error because implementation for  GetNumRejectedTxs not exits.